setup.py

```diff
@@ -48,7 +48,8 @@ def get_long_description():
             "jinja2",
             "python-multipart",
             "pyyaml",
-            "requests",
+            # "httpx>=0.20.0"
+            "httpx @ git+https://github.com/encode/httpx.git@master",
```

Comment:
Until next httpx release. This is a blocker.

Reply:
Could httpx cut a release? This is going to cause issues for anyone that depends on httpx right? I realize you can probably just not install the "full" extra, but this still breaks things for those that are already doing it / it's a bit unergonomic and confusing.